packages/editor/src/worker/worker.ts

```
@@ -1,138 +1,252 @@
+            respondUpdate(newShapes, stats);
+          }
+          break;
+
+        case "CompiledReq":
+          log.info("Received CompiledReq in state Compiled");
+          await compileAndRespond(data);
+          break;
+
+        case "ResampleReq":
+          log.info("Received ResampleReq in state Compiled");
+          const { variation } = data;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)